src/test/java/org/apache/commons/imaging/formats/png/PngWriteForceTrueColorText.java

```diff
@@ -21,21 +21,25 @@
 import org.junit.jupiter.api.Test;
 
 import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
 import java.io.File;
-import java.io.FileOutputStream;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.List;
+import java.util.logging.Logger;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public class PngWriteForceTrueColorText extends PngBaseTest {
+    private static final Logger LOGGER = Logger.getLogger(PngWriteForceTrueColorText.class.getName());
 
     @Test
     public void test() throws Exception {
         final PngImageParser pngImageParser = new PngImageParser();
         final List<File> images = getPngImages();
         for (final File imageFile : images) {
 
+            byte[] outFile = null;
```

Comment:
This could be inside the `try` block. I thought it was declared outside for some reason, but apparently it's only used within that block?